### PR TITLE
Update scaffold chart for TSA

### PR DESCRIPTION
**Pull request by haydentherapper**
Signed-off-by: Hayden Blauzvern <hblauzvern@google.com>

## Checklist

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). Where applicable, update and bump the versions in any associated umbrella chart
- [ ] Variables are documented in the `values.yaml` and added to the README.md. The [helm-docs](https://github.com/norwoodj/helm-docs) utility can be used to generate the necessary content. Use `helm-docs --dry-run` to preview the content.
- [ ] JSON Schema generated.
- [ ] List tests pass for Chart using the [Chart Testing](https://github.com/helm/chart-testing) tool and the `ct lint` command.

### CODE REVIEW

**Comment by haydentherapper** [comment]
I _think_ this is all correct

File: charts/tsa/README.md [hunk 1]
```diff
@@ -95,74 +95,3 @@ To enabled access from external resources, an Ingress resource is created. The c
 >         - host: timestamp.localhost
 >           path: /
 > ```
-
-## Quick Installation
```
Comment (hectorj2f) [inline charts/tsa/README.md:99]: This doesn't look to be right!

Comment (haydentherapper) [reply charts/tsa/README.md:99]: Ah, this is because I need to move this to `README.md.gotmpl`.

**Comment by haydentherapper** [comment]
should be ready to go now

**Comment by haydentherapper** [comment]
@hectorj2f Should be good to go now

TSA version should be updated, but let's get this PR in to avoid more conflicts and then update later

**Comment by haydentherapper** [comment]
@vaikas @hectorj2f bumping for review

File: charts/scaffold/Chart.yaml [hunk 2]
```diff
@@ -35,6 +35,10 @@ dependencies:
     version: 0.1.4
     repository: https://sigstore.github.io/helm-charts
     condition: tuf.enabled
+  - name: tsa
+    version: 0.1.0
+    repository: https://sigstore.github.io/helm-charts
+    condition: tuf.enabled
```
Comment (vaikas) [inline charts/scaffold/Chart.yaml:41]: This seems wrong, shouldn't this be `tsa.enabled`?

**Review by vaikas — COMMENTED** [review]
Yikes, I'm so sorry this fell in the cracks :(

File: charts/scaffold/Chart.yaml (hunk 2 above)
Comment (haydentherapper) [reply charts/scaffold/Chart.yaml]: Fixed!

**Comment by vaikas** [comment]
hm, Chart.lock is conflicting?

**Comment by haydentherapper** [comment]
Whoops, rebased!

**Review by hectorj2f — APPROVED** [review]
lgtm

**Pull request merged**